type-annotator-scanner/build.gradle

```diff
@@ -36,14 +38,21 @@ buildscript {
 plugins {
     id "net.ltgt.errorprone" version "2.0.1" apply false
     id 'com.github.sherter.google-java-format' version '0.9'
+    id 'maven-publish'
```

Comment:
Minor: I don't think we need to add `maven-publish` explicitly if we are already adding `com.vanniktech.maven.publish`?  I think including the latter should include the former automatically.  But I know this is brittle, so if this is working we can leave it this way.

Reply:
Yes this is working for now, Sure I can test it in follow up cycles.